backend/bin/www.js

```diff
@@ -0,0 +1,25 @@
+require('babel-register');
+
+const http = require('http');
+const app = require("../src/app").default;
+
+if (!process.env.DB_CONN_NANOCHAT) {
```

Comment:
As discussed, please document all the necessary steps to start the server locally